schema_converter/cql_to_spanner_schema_converter.go

```diff
@@ -136,11 +140,14 @@ func main() {
 	projectID := flag.String("project", "", "The project ID")
 	instanceID := flag.String("instance", "", "The Spanner instance ID")
 	databaseID := flag.String("database", "", "The Spanner database ID")
+	endpoint := flag.String("endpoint", "", "The Spanner External Host")
 	cqlFile := flag.String("cql", "", "Path to the CQL file")
 	keyspaceFlatter := flag.Bool("keyspaceFlatter", false, "Whether to enable keyspace flattening (default: false)")
 	tableName := flag.String("table", "TableConfigurations", "The name of the table (default: TableConfigurations)")
 	enableUsingTimestamp := flag.Bool("enableUsingTimestamp", false, "Whether to enable using timestamp (default: false)")
 	enableUsingTTL := flag.Bool("enableUsingTTL", false, "Whether to enable TTL (default: false)")
+	usePlainText := flag.Bool("usePlainText", false, "Whether to use plain text to establish connection")
+	ca_certificate_file := flag.String("ca_certificate_file", "", "The CA certificate file to use for TLS")
```

Comment:
I think we need `usePlainText` and `ca_certificate_file` in config.yaml to be use in the proxy.go file? Or I am missing something?